modules/subscriptionService/src/main/scala/mockedSubscriptionResponse/MockedResponse.scala

```diff
@@ -0,0 +1,56 @@
+package mockedSubscriptionResponse
+import io.circe.Json
+import io.circe.parser._
+
+object MockedResponse {
+  def mockedGetUserSubscriptionResponse: Json = {
+    val response =
+      """
+        |{
+        |  "subscriptions":[
+        |    {
+        |      "organization":"47deg",
+        |      "repository":"thool",
+        |      "subscribedAt":"2021-04-15T12:30:15"
+        |    },
+        |    {
+        |      "organization":"higherkindness",
+        |      "repository":"skeuomorph",
+        |      "subscribedAt":"2021-04-26T15:45:25"
+        |    },
+        |    {
+        |      "organization":"47degrees",
+        |      "repository":"github4s",
+        |      "subscribedAt":"2021-05-01T09:15:05"
+        |    }
+        |  ]
+        |}
+        |""".stripMargin
+
+    parse(response).getOrElse(Json.Null)
```

Comment:
I think this would be a perfect use case for using `circe-literal` (`"io.circe" %% "circe-literal" % "0.12.13"`)

```suggestion
    import io.circe.literal._
    json"""
        |{
        |  "subscriptions":[
        |    {
        |      "organization":"47deg",
        |      "repository":"thool",
        |      "subscribedAt":"2021-04-15T12:30:15"
        |    },
        |    {
        |      "organization":"higherkindness",
        |      "repository":"skeuomorph",
        |      "subscribedAt":"2021-04-26T15:45:25"
        |    },
        |    {
        |      "organization":"47degrees",
        |      "repository":"github4s",
        |      "subscribedAt":"2021-05-01T09:15:05"
        |    }
        |  ]
        |}
        |"""
```